File: 0062-unique-paths/0062-unique-paths.cpp
class Solution {
public:
    int uniquePaths(int m, int n) {
        short total = m + n - 2;
        char mind = min(m - 1 , n - 1);
        long long result = 1;
        for(int i = 1 ; i <= mind ; ++i){
            result = result * (total - mind + i) / i ;
        }
        return (int)result;
    }
};

File: 0062-unique-paths/0062-unique-paths.py
class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        c = []
        for i in range(n):
            c.append(1)
        for j in range(m - 1):
            for i in range(1 , n):
                c[i] = c[i - 1] + c[i]
        return c[n - 1]